Model/Rate/Result.php

```diff
@@ -221,14 +271,16 @@ private function createTitle($settingPath)
      * Create price
      * Calculate price if multiple options are chosen
      *
-     * @param $alias
      * @param $settingPath
      *
      * @return float
      */
-    private function createPrice($alias, $settingPath)
+    private function getPrice($settingPath): float
```

Comment:
Je moet in php float zoveel mogelijk vermijden:
https://www.php.net/manual/en/language.types.float.php

So never trust floating number results to the last digit, and do not compare floating point numbers directly for equality.

https://floating-point-gui.de